x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance.go

```diff
@@ -67,7 +67,7 @@ func WithdrawalRewardBalanceCallback(k Keeper, ctx sdk.Context, args []byte, que
 
 	// If there's a rebate portion, fund the community pool with that amount
 	if rebateAmount.GT(sdkmath.ZeroInt()) {
-		rebateToken := sdk.NewCoin(tradeRouteCallback.RewardDenom, rebateAmount)
+		rebateToken := sdk.NewCoin(tradeRoute.RewardDenomOnHostZone, rebateAmount)
```

Comment:
I think these are actually the same thing if you look at how it sets RewardDenom before the call